downloads/swan-lake-release-notes/swan-lake-beta3/RELEASE_NOTE.md

```diff
@@ -0,0 +1,796 @@
+
+```ballerina
+import ballerina/http;
+ 
+service / on new http:Listener(8080) {
+   function init() returns error? {
+       // Return an error for demonstration.
+       // This will result in module initialization failing.
+       return error("Service init failure!");
+   }
+}
+```
+
+#### Using `check` in Object Field Initializers
+
+`check` can now be used in the initializer of an object field if the class or object constructor expression has an `init` method with a compatible return type (i.e., the error type that the expression could evaluate to is a subtype of the return type of the `init` method).
+
+If the expression is used with `check` results in an error value, the `init` method will return the error resulting in either the `new` expression returning an error or the object constructor expression resulting in an error.
+
+```ballerina
+import ballerina/io;
+ 
+int? value = ();
+ 
+class NumberGenerator {
+   int lowerLimit;
+ 
+   // `check` can be used in the field initializer
+   // since the `init` method's return type allows `error`.
+   int upperLimit = check value.ensureType();
+ 
+   function init(int lowerLimit) returns error? {
+       self.lowerLimit = lowerLimit;
+   }
+}
+ 
+public function main() {
+   NumberGenerator|error x = new (0);
+ 
+   if x is error {
+       io:println(x);
+   }
+}
+```
+
+#### Wildcard Binding Pattern Support in Variable Declarations
+
+The wildcard binding pattern can now be used in a variable declaration with a value that belongs to type `any`.
+
+```ballerina
+import ballerina/io;
+ 
+float _ = 3.14;
+var _ = io:println("hello");
+```
+
+Using the wildcard binding pattern when the value is an error will result in a compilation error.
+
+```ballerina
+var _ = error("custom error"); // Compilation error.
+```
+
+#### Relaxed Static Type Requirements for `==` and `!=`
+
+Previously, `==` and `!=` were allowed only when both operands were of static types that are subtypes of `anydata`. This has now been relaxed to allow `==` and `!=` if the static type of at least one operand is a subtype of `anydata`.
+
+```ballerina
+error? x = ();
+ 
+// Now allowed.
+if x == () {
+ 
+}
+``` 
+
+#### Relaxed Query Expression Keywords
+
+Identifiers that are keywords in a query expression context (`where`, `join`, `order`, `by`, `equals`, `ascending`, `descending`, `limit`, `outer`, and `select`) can now be used as ordinary identifiers outside query contexts (i.e., they are no longer required to be quoted identifiers).
+
+```ballerina
+// Now allowed. No longer required to use a quoted identifier (`'limit`).
+int limit = 5;
+// Now allowed. No longer required to use a quoted identifier (`string:'join()`).
+string b = string:join(" ", "hello", "world!"); 
+``` 
+
+### Bug Fixes
+
+- In a stream type `stream<T, C>`; the completion type `C` should always include nil if it is a bounded stream. The bug of this being not validated for stream implementors has been fixed.
```

Comment:
Shouldn't we have the back tick after the semicolon?

Reply:
Yup, semicolon should be within back ticks.